src/parser/priest/shadow/CombatLogParser.js

```diff
@@ -55,10 +56,13 @@ class CombatLogParser extends MainCombatLogParser {
     shadowWordPain: ShadowWordPain,
     voidform: Voidform,
     voidformAverageStacks: VoidformAverageStacks,
-    voidTorrent: VoidTorrent,
     dispersion: Dispersion,
     callToTheVoid: CallToTheVoid,
-    twistOfFate: TwistOfFate,
+
+    // talents:
+    twistOfFate: Talents.talents_45.TwistOfFate,
```

Comment:
This is inconsistent with the rest of the code base and will be awkward should anyone want to overwrite a module in their own parser. I think you should revert this.

Reply:
I've reverted this.